src/main/java/com/deusgmbh/xcusatio/ui/editor/excuse/TextFieldAddBox.java

```diff
@@ -0,0 +1,56 @@
+package com.deusgmbh.xcusatio.ui.editor.excuse;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.control.Button;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.BorderPane;
+
+public class TextFieldAddBox extends BorderPane {
```

Comment:
Bitte kommentar für die Klasse einfügen und erklären was der Sinn dieser Klasse ist und in welchem Kontext sie genutzt werden kann/soll. Aus dem Namen geht das nicht sofort hervor.

Reply:
Und wenn du schon dabei bist, füg auch bei allen anderen Klassen nen kommentar hinzu. Musst nicht überall erklären was genau die Klasse macht, falls der Name schon super offensichtlich ist. Aber zumindest @Author sollte überall stehen ;)

Reply:
hab ich gestern ganz vergessen :D

Reply:
done